### ISSUE TITLE
About the sft dataset selection

### ISSUE THREAD

**Issue by cocoshe** [qizekun/ShapeLLM#12]
in #4 ,
> "For common general understanding tasks, such as VQA, we employ the objaverse dataset for supervised finetuning. You can test this using the gizekun/ShapeLLM_13B_general_v1.0 model. For Grounding tasks, we train using the GAPartNet dataset, and you may select samples from the GAPartNet test set for evaluation."

Select the objaverse dataset as sft dataset for VQA tasks, and select the GAPartNet dataset as sft dataset for Grounding tasks.
So why not do sft for both of the datasets as two sft stages? Is it because the performance may decreases when comparing with the task-specific model on the specific Leaderboard?

**Comment by qizekun** [comment]
Hi,
Yes, having an LLM directly output the 3D bounding box is a highly specific distribution, which can adversely affect performance on general tasks.
